tests/integration/api_v2/http/export.test.js

```diff
@@ -16,26 +16,122 @@
+describe('Export API', () => {
+	const startDate = '2021-01-10';
+	const endDate = '2021-11-30';
+	let refTransaction1;
+	let refTransaction2;
+	beforeAll(async () => {
+		const response1 = await api.get(`${baseUrlV2}/transactions?limit=1`);
+		[refTransaction1] = response1.data;
+
+		const response2 = await api.get(`${baseUrlV2}/transactions?limit=1&offset=1`);
+		[refTransaction2] = response2.data;
+	});
+
+	describe('Schedule file export', () => {
+		it('Schedule file export from account address -> return 202 Accepted', async () => {
+			const expected = { ready: false };
+			const response = await api.get(`${baseUrlV2}/transactions/export?address=${refTransaction1.sender.address}&interval=${startDate}:${endDate}`, 202);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeInstanceOf(Object);
+			expect(response.data).toMap(exportSchemaAccepted);
+			expect(response.meta).toMap(metaSchema);
+			expect(response.meta).toEqual(expect.objectContaining(expected));
+		});
 
-describe('Account history export API', () => {
-	it('return csv file -> 200 OK', async () => {
-		const validFile = 'test.csv';
-		const response = await api.get(`${endpoint}/${validFile}`, 200);
-		expect(response).toEqual(expectedResponse);
+		it('Schedule file export from account publicKey -> return 202 Accepted', async () => {
+			const expected = { ready: false };
+			const response = await api.get(`${baseUrlV2}/transactions/export?publicKey=${refTransaction2.sender.publicKey}&interval=${startDate}:${endDate}`, 202);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeInstanceOf(Object);
+			expect(response.data).toMap(exportSchemaAccepted);
+			expect(response.meta).toMap(metaSchema);
+			expect(response.meta).toEqual(expect.objectContaining(expected));
+		});
 	});
 
-	it('File not exists -> 404 NOT_FOUND', async () => {
-		const invalidFile = 'invalid.csv';
-		const response = await api.get(`${endpoint}/${invalidFile}`, 404);
-		expect(response).toMap(notFoundSchema);
+	describe('File is ready to export', () => {
+		it('scheduled from account address -> return 200 OK', async () => {
+			// Add delay of 10 seconds
+			await new Promise(resolve => setTimeout(resolve, 10000));
```

Comment:
Did you consider adding a setInterval and wait for the 200 OK response in 500ms intervals?
A helper function that resolves only if the result is correct. Otherwise it hits the timeout of the tests.